packages/salesforcedx-sobjects-faux-generator/src/transformer/transformerFactory.ts

```diff
@@ -88,12 +86,11 @@ export class SObjectTransformerFactory {
     );
   }
 
-  public static async createConnection(
-    projectPath: string
-  ): Promise<Connection> {
+  public static async createConnection(): Promise<Connection> {
+    // TODO: replace below logic with a call to a common function for getting a connection.
     const connection = await Connection.create({
       authInfo: await AuthInfo.create({
-        username: await ConfigUtil.getUsername(projectPath)
+        username: await ConfigUtil.getUsername()
```

Comment:
Should the TODO be handled right now by importing WCU from utils? ie: 
`await WorkspaceContextUtil.getInstance().getConnection();`

Reply:
good call...I'll give it a shot